tests/integration/cmor/_fixes/cmip6/test_mcm_ua_1_0.py

```diff
@@ -176,6 +177,7 @@ def test_allvars_fix_lon_bounds(cubes_bounds):
             pass
         else:
             assert lon_coord.bounds[-1][-1] == 358.125
+            assert lon_coord.circular is True
```

Comment:
Splitting hairs, it might be simpler to say
```suggestion
            assert lon_coord.circular
```